packaging/linux/deb/package_binaries.sh

```diff
@@ -30,8 +30,10 @@ if [ "$mode" = "production" ] ; then
   repo_url="http://dist.keybase.io/linux/deb/repo"
 elif [ "$mode" = "prerelease" ] ; then
   repo_url="http://prerelease.keybase.io/deb"
-  # "psmisc" provides "killall", which is used in run_keybase.
-  dependencies="Depends: libappindicator1, fuse, libgconf-2-4, psmisc"
+  # * "psmisc" provides "killall", which is used in run_keybase and
+  #   post_install.sh.
+  # * "coreutils" provides "nohup", which is used in run_keybase.
+  dependencies="Depends: libappindicator1, fuse, libgconf-2-4, psmisc, coreutils"
```

Comment:
Violates https://www.debian.org/doc/debian-policy/#dependencies

Reply:
Are you referring to the item about specifying "essential" dependencies? Could you clarify?

I assume that we run into problems in some Docker environments if we don't make this dependency explicit? Is that right @strib?

Reply:
> I assume that we run into problems in some Docker environments if we don't make this dependency explicit. Is that right @strib?

That was a lesson I learned with the `psmisc` dependency for `killall`.  For this one I just proactively added the dependency.  I am ok removing it.  Either way.

Reply:
I'd say if we haven't hit a case yet where `nohup` is missing, then let's leave it out. `coreutils` is the same package that provides `cat` and `mkdir` and all the other essentials after all. (Maybe there are systems where `busybox` provides these things instead of `coreutils`, but then `busybox` provides `nohup` too.)